waspc/src/Generator/ServerGenerator/OperationsRoutesG.hs

```diff
@@ -79,25 +79,34 @@ relPosixPathFromOperationsRoutesDirToSrcDir :: Path Posix (Rel OperationsRoutesD
 relPosixPathFromOperationsRoutesDirToSrcDir = [reldirP|../..|]
 
 genOperationsRouter :: Wasp -> FileDraft
-genOperationsRouter wasp = C.makeTemplateFD tmplFile dstFile (Just tmplData)
+genOperationsRouter wasp
+   -- TODO: Move below check to Analyzer and make a semantic check
```

Comment:
```suggestion
   -- TODO: Right now we are throwing error here, but we should instead perform this check in parsing/analyzer phase, as a semantic check, since we have all the info we need then already.
```

Reply:
done